### PR TITLE
Don't show last Comma in hosts lists at target details

### PR DESCRIPTION
**Pull request by swaterkamp**
Also fix small typo

**Checklist**:

- [] Tests
- [] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/pages/alerts/details.js [hunk 1]
```diff
@@ -147,10 +147,10 @@ const AlertDetails = ({capabilities, entity, links = true}) => {
 
           {tasks.length > 0 && (
             <TableRow>
-              <TableData>{_('Task using this Alert')}</TableData>
+              <TableData>{_('Tasks using this Alert')}</TableData>
               <TableData>
                 <HorizontalSep wrap>
-                  {tasks.map(task => (
+                  {tasks.map((task, index) => (
```
Comment (bjoernricks) [inline gsa/src/web/pages/alerts/details.js]: Why did you add the unused index here?

Comment (swaterkamp) [reply gsa/src/web/pages/alerts/details.js]: It was an accident... I forgot to remove it.

**Comment by swaterkamp** [comment]
> Could you check if something like
> 
> ```js-jsx
> <HorizontalSep seperator="," wrap>....map(host => <span>{host}</span>)</HorizontalSep>
> ```
> 
> could fit here too?

I tried the HorizontalSep and it's not optimal for this use case. It's more useful for separating items with a centered item and not a left-bound one. Also the whole string got shifted to the left as well, leading to a broken left-alignment of information.

**Comment by swaterkamp** [comment]
> just use
> 
> ```diff
> [...]
> ```

I didn't have in mind to actually change the HorizontalSep. Done now, looks good.
